src/ol/control/attributioncontrol.js

```diff
@@ -244,7 +253,9 @@ ol.control.Attribution.prototype.updateElement_ = function(frameState) {
       }
       delete hiddenAttributions[attributionKey];
     } else {
-      goog.dom.removeNode(this.attributionElements_[attributionKey]);
+      if (this.attributionElements_[attributionKey] && this.attributionElements_[attributionKey].parentNode) {
```

Comment:
Maybe assign `this.attributionElements_[attributionKey]` to avoid repeated dereferencing.